modules/wildfly/pom.xml

```diff
@@ -61,11 +61,6 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
```

Comment:
This is only the declaration of the dependency, we are not using it in the assembly that creates the module for WIldFly.
WildFly already contains hibernate-infinispan in the ORM module.

The only issue I can see is that WildFly 8.2 contains ORM 4.3.7 instead of 4.3.8 but these does not seems to cause problems at the moment.

This does not seem to cause issues, I would wait until they upgrade the dependencies in WIldFly